rules/integrations/aws/credential_access_new_terms_secretsmanager_getsecretvalue.toml

```diff
@@ -53,6 +53,7 @@ This rule looks for the retrieval of credentials using `GetSecretValue` action i
 
 ### False positive analysis
 
+- Review `user.id` values for expected ARNs. If this is an expected behavior, consider adding exceptions to the rule.
```

Comment:
Note that while this is a relatively small change, we note why we are unable to tune the rule at this time in the tuning meta. As a result, we want to encourage user's to add exceptions on `user.id` to reduce FPs.